Sprint_6

TimYandexProfile opened the pull request:
Добавлены тесты из обязательного задания.
Добавлены моки.
Применена параметризация.
Добавлен JaCoCo.

TimYandexProfile committed:
Добавлены тесты из обязательного задания.
Добавлены моки.
Применена параметризация.
Добавлен JaCoCo.

alesyaisachenko commented on .idea/compiler.xml:
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Это нужно доработать:
папку .idea нужно добавлять в .gitignore

alesyaisachenko commented on src/test/java/AnimalTest.java:
```diff
@@ -0,0 +1,46 @@
+    private static final String PREDATOR = "Хищник";
+    private static final String EXPECTED_FAMILIES = "Существует несколько семейств: заячьи, беличьи, мышиные, кошачьи, псовые, медвежьи, куньи";
+
+    @InjectMocks
```
Это нужно доработать:
здесь аннотация @InjectMocks лишняя, ее нужно удалить

alesyaisachenko commented on src/test/java/FelineTest.java:
```diff
@@ -0,0 +1,43 @@
+public class FelineTest {
+
+    public static final int KITTENS_COUNT = 3;
+    @InjectMocks
```
Это нужно доработать:
здесь аннотация @InjectMocks лишняя, ее нужно удалить

TimYandexProfile committed:
Спасибо) Делал не без помощи более опытных людей.
Убрал аннотации в тестах AnimalTest.java и FelineTest.java
Добавил .idea в .gitignore
Добавил скрин отчёта в qa_java
На другом компьютере тесты проходят, отчёт собирается, на этом постоянно ругается surefire плагин. Убирал его с помощью добавления игнорирования в pom.xml, добавлял и пробовал разные настройки, каждый раз при clean и install в Lifecycle такое. Поэтому остаётся показать только скрином.
